dev-lisp/sbcl/sbcl-2.2.4.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="git://github.com/sbcl/sbcl.git"
+CHECKSUM_SHA256="9cfd324f2a1eac2635188b945d10df428d1941c6"
+PATCHES="sbcl-2.2.4.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
```

Comment:
It doesn't appear that these variables are being used in the recipe.

Reply:
updated